src/screens/Settings/components/OtherSettings/index.tsx

```diff
@@ -280,12 +312,43 @@ export default function OtherSettings({
         </SelectField>
       )}
       {!isWin && (
-        <TextInputField
+        <TableInput
           label={t('options.advanced.title')}
-          htmlId="otherOptions"
-          placeholder={t('options.advanced.placeholder')}
-          value={otherOptions}
-          onChange={handleOtherOptions}
+          htmlId={'enviromentOptions'}
+          header={{
+            key: t('options.advanced.key', 'Key'),
+            value: t('options.advanced.value', 'Value')
+          }}
+          rows={getEnvironmentVariables()}
+          onChange={handleEnviromentVariables}
+          inputPlaceHolder={{
+            key: t('options.advanced.placeHolderKey', 'ENVIORMENT'),
+            value: t(
+              'options.advanced.placeHolderValue',
+              'E.g.: Path/To/ExtraFiles'
+            )
+          }}
+          afterInput={info}
+        />
+      )}
+      {!isWin && (
+        <TableInput
+          label={t('options.wrapper.title', 'Wrapper additional arguments')}
```

Comment:
I'm confused since I've never used the previous wrapper option, but how does multiple wrappers work? I imagine it is order-dependent so maybe it should be specified somewhere (using the `info` block) that the wrappers are applied in the order defined in the table?

Reply:
I also never used wrapper. I thin if someone uses it it should be clear to them.